library/data/registry_entry_attachments/src/main/java/org/quiltmc/qsl/registry/attachment/api/RegistryEntryAttachment.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 QuiltMC
+ * Copyright 2022 QuiltMC
```

Comment:
wait aaaaaaa i missed one
```suggestion
 * Copyright 2021-2022 QuiltMC
```